e2e/test/Common/EventSourceTestLogger.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    // f7ac322b-77f1-5a2d-0b56-ec79a41e82a2
+    [EventSource(Name = "Microsoft-Azure-Devices-TestLogging")]
+    internal class EventSourceTestLogger : EventSource
```

Comment:
don't unit test files use this class as well?

Reply:
Everything builds after removing it. Plus I looked at all the .csproj for the links and did not find anything.